### DISCUSSION TITLE
Login in iFrame not possible with Chromium-based browser

### DISCUSSION THREAD

**Discussion by davidkopp** [requarks/wiki#5912] [Error / Bug Report]
If Wiki.js is embedded as an iFrame, it is not possible to login with a Chromium-based browser. Instead of a successful login, you are redirected to `/login`. This problem does not exist with other browsers.
I suspect that the reason for this is the cookie attribute `SameSite`. See for example this Stack Overflow answer: https://stackoverflow.com/a/45095345

My test page:
```html
 <!DOCTYPE html>
<html>
<head></head>
<body>
 <iframe src="https://wiki.cyclize.de" title="Cyclize Wiki" height="800px" width="800px"></iframe> 
</body>
</html>  
```

Wiki.js version: 2.5.292 (latest)

Background:
I would like to embed Wiki.js in my [ClickUp](https://clickup.com/) workspace (as an iFrame).

**Comment by NGPixel** [comment]
Did you disable the iframe embedding protection under Admin > Security?

**Reply by davidkopp** [reply]
yes, the option "Block IFrame Embedding" is disabled. As I said, the problem only exists in Chromium-based browsers, not with Mozilla Firefox, for example.

**Comment by ReessKennedy** [comment]
WIkiJS is awesome but +1 on this issue as well .... I am frustrated by it. 

I was so excited when I saw the options to toggle off XFRAME to allow for it right inside WIkiJS thinking, "this is the solution I can use in Clickup" and got somewhat invested in setting up WikiJS and now I am realizing it actually won't work with Chrome Clickup.

**Comment by julianbermolen** [comment]
have any answer or solution about this problem ?

**Reply by srd90** [reply]
Browser vendors started process of blocking by default 3rd party cookies (affects especially/among other things content in iframes) years ago. First by introducing samesite settings (default lax) etc:
* https://chromestatus.com/feature/5088147346030592
* https://web.dev/articles/samesite-cookies-explained

It was possible with e.g. Chrome to workaround with samesite none and secure https://blog.chromium.org/2019/10/developers-get-ready-for-new.html

E.g. Webkit took stricter approach around 2020 https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/

More recent (Dec 2023 and Jan 2024) blogs about status of 3rd party cookie blocking rollout

https://developer.mozilla.org/en-US/blog/goodbye-third-party-cookies/

https://developers.google.com/privacy-sandbox/blog/cookie-countdown-2024jan

So...as long as wikijs is using/if it is using cookies to track login session (and therefore use information stored to sessions designated by cookies to authorize access from UI to wikijs backend APIs) you should not expect to be able to use autenticated access to wikijs that is embedded to iframe.
